README.md

```diff
@@ -221,3 +224,81 @@ Depending on the input parameters, the SWS may decide one of the following:
+
+- imagine a robot doing work on a construction site
+- while moving, the robot detects an unexpected obstacle in its way (e.g. some construction materials that are stored there temporarily)
+- depending on the situation, either of the following may now happen:
+  - the robot may be able to autonomously circumnavigate the obstacle and continue its work
+  - the robot may be blocked and thus unable to continue its work, so that it requires human support
+- depending on the situation, SWS should support the decision if/when a human should be called for help.
+
+The SWS management should decide if/when a human should be called to help the robot. This decision depends on:
+
+- the time that the robot is already blocked
+- the battery status of the robot
+- the expected waiting time until the human will come to help
+
+#### Example Rules
+
+- if the battery status of robot is low, ask for human help
+- if the time of robot blocked is short and battery status of robot is not low, let robot continue trying
+- if the time of robot blocked is moderate and battery status of robot is not low, Inform human about the problem
+
+To explore the example rules in detail, please refer to the FCL file located [here](src/main/resources/rules/ConstructionRobotAssistanceDecisionRules.fcl)
+
+#### How to Run and Test the Scenario
+
+To start the application and run the construction scenario, use the following command:
+
+```bash
+mvn spring-boot:run -D"spring-boot.run.profiles"=construction
+```
+
+##### Example Request
+
+Execute the following `curl` command in your terminal to request a "sliding decision" via a POST request to the `/sliding-decision` endpoint:
+
+```bash
+curl --request POST \
+  --url http://localhost:8080/sliding-decision \
+  --header "Content-Type: application/json" \
+  --data '{
+    "decisionStatus": "Sliding Decision Request",
+    "slidingDecisionInputParameters": {
+      "timeRobotIsBlocked": 4,
+      "robotBatteryStatus": 65,
+      "waitingTimeForHuman": 10
+    }
+  }'
+```
+
+To test the implemented rules, you can pass different inputs to the application and observe the outcomes. Just modify the values for the `slidingDecisionInputParameters` as follows:
+- `timeRobotIsBlocked`: The time that the robot is blocked, measured in minutes (0-15 minutes)
+- `robotBatteryStatus`: The battery status of the robot, measured in percent (0%-100%)
+- `waitingTimeForHuman`: The waiting time for the human to become available, measured in minutes (0-15 minutes)
+
+##### Example Response
+
+The application will respond with a JSON string similar to the following:
+
+```json
+{
+  "decisionStatus": "Sliding Decision Response",
+  "decisionResult": {
+    "slidingDecision": "AI_AUTONOMOUSLY",
+    "description": "Let the robot continue trying"
+  }
+}
+```
+
+Depending on the input, the SWS may decide one of the following:
+- `AI_AUTONOMOUSLY`: "Let the robot continue trying"
+- `HUMAN_IN_THE_LOOP`: "Warn human, but let them decide"
```

Comment:
I would move this line one down, so that the lines are ordered "from highest to lowest degree of robot autonomy". Also I would make it a little bit more clear:
```suggestion
- `HUMAN_IN_THE_LOOP`: "Warn human, but let them decide if/when to help"
```